app/views/posts/show.html.erb

```diff
@@ -1,3 +1,8 @@
-<h1>
-    This is the show page for posts
-</h1>
\ No newline at end of file
+<div class="top-com">
+    <h1>Post#<%= @post.id %> by <%= @user.name %></h1>
+    <%= render partial: 'users/show', object: @post, as: :post %>
+</div>
+<div class="comment-container">
+    <h3>Post Comments</h3>
+    <%= show_comments(@post) %>
+</div>
```

Comment:
- [ ] We should have a button to see all the posts on the user's show page. This button should take users to all the posts for that user(for example, the first user in this project)

![recent](https://github.com/Shakir-Hussain12/blog_app/assets/53124548/cad37c65-bfb8-4024-af79-8a79855915d9)